examples/robolectricExample/src/test/java/io/realm/examples/robolectric/RobolectricTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.robolectric;
+
+import android.app.Activity;
+import android.content.Context;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.annotation.Config;
+import org.robolectric.shadows.ShadowLog;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.realm.Realm;
+import io.realm.RealmConfiguration;
+import io.realm.examples.robolectric.entities.Person;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(RobolectricGradleTestRunner.class)
+@Config(constants = BuildConfig.class)
+public class RobolectricTest {
+    protected final static int TEST_DATA_SIZE = 10;
+
+    protected List<String> columnData = new ArrayList<String>();
+
+    private final static String FIELD_STRING = "columnString";
+    private final static String FIELD_LONG = "columnLong";
+    private final static String FIELD_FLOAT = "columnFloat";
+    private final static String FIELD_DOUBLE = "columnDouble";
+    private final static String FIELD_BOOLEAN = "columnBoolean";
+    private final static String FIELD_DATE = "columnDate";
+
+    private Realm testRealm;
+    private Activity context;
+    private RealmConfiguration testConfig;
+
+    private Context getContext() {
+        return context;
+    }
+
+    @BeforeClass
+    public static void beforeClassSetUp() {
+        System.setProperty("java.library.path", "./robolectricLibs");
```

Comment:
Well, if the file can just be placed in `test/libs` We don't have to do anything right? But yes otherwise it should just be a helper class.